Helios/Interfaces/Profile/ProfileInterface.cs

```diff
@@ -37,6 +39,14 @@ public class ProfileInterface : HeliosInterface, IStatusReportNotify, IResetMoni
         // our implementation of IStatusReportNotify
         private readonly StatusReportNotifyAsyncOnce _statusReportNotify;
 
+        // definition for profile related triggers
+        private HeliosTrigger _profileStartedTrigger;
+        private HeliosTrigger _profileResetTrigger;
+        private HeliosTrigger _profileStoppedTrigger;
+
+        private static Regex _rxDoubleQuotes = new Regex("(?:\\\")([^\"]*)(?:\\\")", RegexOptions.IgnoreCase | RegexOptions.IgnorePatternWhitespace);
+        private static Regex _rxExe = new Regex("(" + Environment.GetEnvironmentVariable("PATHEXT").Replace(";", "|").Replace(".", "\\.") + ")", RegexOptions.IgnoreCase | RegexOptions.IgnorePatternWhitespace);
```

Comment:
GetEnvironmentVariable can return null.   so you must include something like 

GetEnvironmentVariable("PATHEXT")?.Replace(bla bla bla) ?? "" 

to gracefully handle Null, particularly since this is executed at an unknown time (static init whenever) which is a super nasty thing to debug

Reply:
null handled.